pkg/regexp/regexp.go

```diff
@@ -0,0 +1,214 @@
+package regexp
+
+import (
+	"io"
+	"regexp"
+	"sync"
+)
+
+// Regexp is a wrapper struct used for wrapping MustCompile regex expressions
+// used as global variables. Using this stucture helps speed the startup time
+// of apps that want to use global regex variable. This librry initialized on
```

Comment:
```suggestion
// of apps that want to use global regex variables. This library initializes them on
```

And possibly “… that want to use global regex variables, especially with containers/storage/pkg/reexec.“